components/channel_header/components/header_icon_wrapper.jsx

```diff
@@ -61,7 +62,20 @@ export default function HeaderIconWrapper({
         );
     }
 
-    const tooltip = getTooltip(tooltipKey);
+    let tooltip;
+    if (tooltipKey === 'plugin' && tooltipText && tooltipText !== '') {
```

Comment:
Note that `tooltipText && tooltipText !== ''` is redundant: `tooltipText` will be truthy iff it's non-empty.

Reply:
Thanks 👍